libs/akita/src/lib/entitiesToMap.ts

```diff
@@ -15,29 +16,30 @@ export function entitiesToMap<S extends EntityState<E>, E>(state: S, options) {
 
   if (filterBy && hasLimit) {
     let count = 0;
-    for (let i = 0, length = ids.length; i < length; i++) {
+    for (let i = 0, { length } = ids; i < length; i += 1) {
       if (count === limitTo) break;
       const id = ids[i];
       const entity = entities[id];
-      const allPass = coerceArray(filterBy).every(fn => fn(entity, i));
+      const allPass = coerceArray(filterBy).every((fn) => fn(entity, i));
       if (allPass) {
         map[id] = entity;
-        count++;
+        count += 1;
```

Comment:
why this change?

Reply:
Ye I wasn't super happy with this either, but their reasoning seems sound
https://github.com/airbnb/javascript#variables--unary-increment-decrement

Reply:
I'm going to revert this rule, as it's quite annoying and I believe TS+prettier will guard against the pit falls this rule tries to prevent

Reply:
Great